### DISCUSSION TITLE
Release 1.0.0 MVP

### DISCUSSION THREAD

**Discussion by xxfast** [xxfast/fabric-witness#7] [Ideas]
`fabric-witness` mod is a huge project! I eventually want to make a full witness experience in Minecraft. Unfortunately, I don't have all the time in the world to work on this project. Therefore, I will need to prioritise features for the first `1.0.0` release, and this is the purpose of this discussion thread. 

These are the features I'm planning for the [first milestone](https://github.com/xxfast/fabric-witness/milestone/1)
- [ ] Add basic puzzle solving #5 
- [ ] Add Hexagon dots #4 
- [ ] Make solved puzzle frames emit a Redstone signal #6

**Comment by WhatYouFind** [comment]
Yo this looks really cool! Are you still working on it?
